### DISCUSSION TITLE
What to do with ENEX files?

### DISCUSSION THREAD

**Discussion by sergeevabc** [vzhd1701/evernote-backup#68] [Q&A] [answered]
`en_backup.db` is SQLite database, `something.enex` looks like XML.

This is what you get on disk by following the instructions of this program. But nothing is said about what to do next. After all, you cannot open *.enex, like an archive or a web page with text and pictures.

**Comment by thesamim** [comment, marked as answer]
You can import them into any number of other applications. Personally: I use Obsidian and the import evernote plugin.
